.github/stock exchange1&2/script2.js

```diff
@@ -0,0 +1,28 @@
+const urlArr = window.location.href.split('=');
+
+const symbol = urlArr[urlArr.length - 1];
```

Comment:
although this way of extracting the symbol from the url works, it's better to use the URLSearchParams object like suggested in the project instractions